bunsen/bunsen-core-r4/src/main/java/com/cerner/bunsen/definitions/r4/R4StructureDefinitions.java

```diff
@@ -181,7 +188,7 @@ private <T> List<StructureField<T>> extensionElementToFields(DefinitionVisitor<T
           rootDefinition,
           element.getSliceName(),
           stack,
-          getFirstProfile(element.getTypeFirstRep().getProfile()),
+          getFirstTypeProfile(element),
```

Comment:
can we use the `profileUrl` variable that was made using the same function here?